app/assets/stylesheets/views/_browse.scss

```diff
@@ -92,4 +92,18 @@ $browse-header-background-colour: #263135;
   @include govuk-media-query($from: "tablet") {
     margin-bottom: govuk-spacing(6);
   }
-}
\ No newline at end of file
+}
+
+.browse__action-links .gem-c-chat-entry {
+  @include govuk-media-query($until: "tablet") {
+    border-top: 4px solid govuk-colour("blue");
+    padding-top: govuk-spacing(5);
+  }
+}
+
+.browse__govuk-chat-promo .gem-c-chat-entry {
+  @include govuk-media-query($until: "tablet") {
+    border-top: 2px solid govuk-colour("blue");
+    padding-top: govuk-spacing(5);
+  }
```

Comment:
I think we are committing a sin by modifying the component directly because of the tight coupling this produces. (I tried to find a doc but best I got was: https://github.com/alphagov/govuk_publishing_components/blob/main/docs/component_principles.md which doesn't cover this point precisely).

I do think it makes sense to be doing this styling change in this app rather than the component though since this is specific to the browse page and wouldn't make sense decoupled.

I think instead we should use a wrapper div to apply the styles e.g:

```
.browse__govuk-chat-promo {
  @include govuk-media-query($until: "tablet") {
    border-top: 2px solid govuk-colour("blue");
    padding-top: govuk-spacing(5);
  }
```

and then we change the markup for both of the additions for HTML i.e:

```
<% if show_govuk_chat_promo?(page.base_path) %>
  <div class="govuk-grid-column-one-third">
    <div class="browse__govuk-chat-promo">
      <%= render "govuk_publishing_components/components/chat_entry" %>
    </div>
  </div>
<% end %>
```

I expect we could use that same class for both browse usages.

Reply:
Argh 🙈 did not spot that minor difference. Thanks for spotting.

Change seems good, will have another suggestion though.